plugins/tedge_apt_plugin/Cargo.toml

```diff
@@ -10,7 +10,7 @@ homepage = { workspace = true }
 repository = { workspace = true }
 
 [dependencies]
-clap = { version = "3", features = ["derive"] }
+clap = { workspace = true, features = ["derive"] }
```

Comment:
The system test errors are likely related to this change: https://github.com/thin-edge/thin-edge.io/actions/runs/6039139197/job/16387472405?pr=2206.

```
`sudo /etc/tedge/sm-plugins/apt...` returned an unexpected exit code

  error: unexpected argument '--name' found

  tip: a similar argument exists: '----name'

Usage: apt list <----name <NAME>|----maintainer <MAINTAINER>>
```

Reply:
Yes, I fixed it.